app/controllers/admin/tenants_controller.rb

```diff
@@ -1,5 +1,58 @@
 class Admin::TenantsController < Admin::BaseController
   def index
-    @tenants = Tenant.select(:organization, :location, :slug).all
+    if !Tenant.unapproved.empty?
+      @unapproved = Tenant.unapproved.select(:organization,
+                                             :location,
+                                             :slug,
+                                             :id).all.order(:id)
+    else
+      @unapproved = nil
+    end
+
+    if !Tenant.inactive.empty?
+      @inactive = Tenant.inactive.select(:organization,
+                                         :location,
+                                         :slug,
+                                         :id).all.order(:id)
+    else
+      @inactive = nil
+    end
+
+    if !Tenant.actives.empty?
+      @actives = Tenant.actives.select(:organization,
+                                       :location,
+                                       :slug,
+                                       :id).all.order(:id)
+    else
+      @actives = nil
+    end
+  end
```

Comment:
This seems really repetitive...off-hand I'm not thinking of a better way, but all three if statements have exactly the same format. Maybe make an array of statuses and loop through?